content/md/articles/cookbooks/cli_build_projects.md

```diff
@@ -772,3 +772,48 @@ and calls `jar` which uses
 [both `with-dir` and `with-project-root`](https://github.com/polyfy/polylith/blob/9053b190d5f3b0680ac4fe5c5f1851f7c0d40830/build.clj#L146-L147)
 to set the project root while performing `tools.deps` and `tools.build`
 operations.
+
+## Including java code in a Clojure project
+Many Clojure projects will, at some point, require the inclusion of Java files.
+Adding and using such files is simple and can be done like this:
+First, run `mkdir -p java/src/mypackage` in the root of your Clojure project.
```

Comment:
Add a qualifying phrase like "where `mypackage` is the package name your Java code will use." to make it clear that this isn't a literal string but something user-driven.

Reply:
Done, thank you